hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -50,6 +50,8 @@ public record TokensConfig(
                 boolean nftsUseVirtualMerkle,
         @ConfigProperty(value = "maxAllowedAirdrops", defaultValue = "1000000") @NetworkProperty
                 long maxAllowedAirdrops,
+        @ConfigProperty(value = "maxAllowedAirdropTransfers", defaultValue = "10") @NetworkProperty
+                long maxAllowedAirdropTransfers,
```

Comment:
Why do we need this?

Reply:
Similar to this discussion https://github.com/hashgraph/hedera-services/pull/14379#discussion_r1697061341 - I am moving this value to a config.

I'm not sure if we want to have 3 different configurations for this or only one. Maybe here is the misunderstanding?